### ISSUE TITLE
Fix all the TODO's

### ISSUE THREAD

**Issue by benjie** [graphile/graphile.github.io#13]
This issue will probably never close; it's a perennial reminder that the TODOs in the documentation need addressing.

Help doing this is very welcome; you can **just fix them one at a time if you like**!

Some things are simple "check this example works" - so if you can check it and it works you can delete the text and submit that as a PR 👍 

Others are adding examples - which you'd also be very welcome to submit as a PR 👍 

Basically what I'm trying to say is:

# PRs welcome!
